### PR TITLE
Add Download BAM vignette to CI (closes #32)

### LINKED ISSUE #32: Add download BAM vignette test to CI harness

### PR DESCRIPTION
**Pull request by natanlao**
This PR updates the Download BAM notebook so that it passes CI (i.e. so that every cell runs without error). This generally involves catching and explaining cells running code expected to fail.

I've also made a clarity tweak: given a `bundle_fqid`, I think that `bundle_uuid, bundle_version = bundle_fqid.split('.', maxsplit=1)` is clearer than `bundle_uuid = bundle_fqid[:36]`.

### CODE REVIEW

File: tasks/Download BAM/Download Any BAM.ipynb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   },
```
Comment (amarjandu) [inline tasks/Download BAM/Download Any BAM.ipynb]: <p>are we supposed to be able to see the output?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (amarjandu) [reply tasks/Download BAM/Download Any BAM.ipynb]: It looks good, is this supposed to have output data in the journal? or is the user supposed to run it themselves and follow along? @mckinsel i know its been a while sense you have modified the file, any input?

If they are to be following along i think it makes sense to test if this file contains output; so the next time someone updates it they don't commit the history, if its to be omitted. @natanlao

Comment (natanlao) [reply tasks/Download BAM/Download Any BAM.ipynb]: Okay, then I'll update this notebook to include output, and then file a separate PR to check for notebook output in CI. Does that sound ok?

**Comment by natanlao** [comment]
I spoke with @amarjandu in person, who advised me that I should add output to this notebook then merge.

**Pull request merged**